compiler/rustc_builtin_macros/src/compile_error.rs

```diff
@@ -1,22 +1,28 @@
 // The compiler code necessary to support the compile_error! extension.
 
 use rustc_ast::tokenstream::TokenStream;
-use rustc_expand::base::{get_single_str_from_tts, DummyResult, ExtCtxt, MacResult};
+use rustc_expand::base::{
+    get_single_str_from_tts, DummyResult, ExpandResult, ExtCtxt, MacroExpanderResult,
+};
```

Comment:
Style nit: could you keep the single-line import style here and in other places?